okio-fakefilesystem/build.gradle.kts

```diff
@@ -10,7 +10,7 @@ kotlin {
     withJava()
   }
   if (kmpJsEnabled) {
-    js {
+    js(BOTH) {
```

Comment:
You can specify this once in the root `gradle.properties` as
```properties
kotlin.js.compiler=both
```
rather than having to remember to update all modules.

Reply:
Fixed!